mygnupg.py

```diff
@@ -0,0 +1,64 @@
+import gnupg
+
+class mygnupg():
+    """
+    This class used for initial gnupg, 
+    create, import, and export key pairs.
+    Also, can encrypt and decrypt data.
+    """
+    def __init__(self):
+        """
+        The binary file should be different
+        in different os.
+        This is MacOS version.
+        """
+        self.gpg = gnupg.GPG(gnupghome='./gnupgkeys')
+        self.key = None
+        self.email = "john_smith@scmail.org"
```

Comment:
The constructor for this class should probably take some optional (with defaults) arguments for these variables. For instance, the email and password will probably be passed on the command line or pulled from the environment and then given to an instance of this class.

Reply:
Thanks, I already added this part.